examples/node/cluster_example.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+import {
+    ClosingError,
+    ConnectionError,
+    GlideClusterClient,
+    InfoOptions,
+    Logger,
+    RequestError,
+    TimeoutError,
+} from "@valkey/valkey-glide";
+
+/**
+ * Creates and returns a GlideClusterClient instance.
+ * This function initializes a GlideClusterClient with the provided list of nodes.
+ * The nodesList may contain the address of one or more cluster nodes, and the
+ * client will automatically discover all nodes in the cluster.
+ * @param nodesList A list of tuples where each tuple contains a host (str) and port (int). Defaults to [("localhost", 6379)].
+ * @returns An instance of GlideClusterClient connected to the discovered nodes.
+ */
+async function createClient(nodesList = [{ host: "localhost", port: 6379 }]) {
+    const addresses = nodesList.map((node) => ({
+        host: node.host,
+        port: node.port,
+    }));
+
+    // Check `GlideClusterClientConfiguration` for additional options.
+    return await GlideClusterClient.createClient({
+        addresses: addresses,
+        // if the server uses TLS, you'll need to enable it. Otherwise the connection attempt will time out silently.
+        // useTLS: true,
+    });
+}
+
+/**
+ * Executes the main logic of the application, performing basic operations
+ * such as SET, GET, PING, and INFO REPLICATION using the provided GlideClusterClient.
+ * @param client An instance of GlideClusterClient.
+ */
+async function appLogic(client: GlideClusterClient) {
+    // Send SET and GET
+    const setResponse = await client.set("foo", "bar");
+    Logger.log("info", "app", `Set response is: ${setResponse}`);
+
+    const getResponse = await client.get("foo");
+    Logger.log("info", "app", `Get response is: ${getResponse?.toString()}`);
+
+    // Send PING to all primaries (according to Redis's PING request_policy)
```

Comment:
```suggestion
    // Send PING, by default it is sent to all primary nodes
```
an end user shouldn't know about client internals

Reply:
LGTM, In addition it is in sync with Python